packages/core/fatal-errors/core-fatal-errors-browser-internal/src/fatal_errors_screen.tsx

```diff
@@ -17,17 +17,16 @@ import {
   EuiPageContent,
 } from '@elastic/eui';
 import React from 'react';
-import * as Rx from 'rxjs';
-import { tap } from 'rxjs/operators';
+import { Observable, Subscription, merge, tap, fromEvent } from 'rxjs';
```

Comment:
I changed the imports a bit to get around an issue with using `* as` syntax. Should I have rather done:
```suggestion
import { Observable, Subscription, merge, fromEvent } from 'rxjs';
import { tap } from 'rxjs/operators';
```
?